docs/cli/csv-parser-state.md

```diff
@@ -0,0 +1,163 @@
+
+Options:
+  --help, -h                 Show help                                 [boolean]
+  --version, -v              Show version number                       [boolean]
+  --projectKey, -p           API project key                            [string]
+  --apiUrl                   The host URL of the HTTP API service
+                             [string] [default: "https://api.commercetools.com"]
+  --authUrl                  The host URL of the OAuth API service
+                            [string] [default: "https://auth.commercetools.com"]
+  --accessToken              CTP client access token
+                             Required scopes: ['view_orders']         [string]
+  --input, -i                Path to CSV file.                [default: "stdin"]
+  --output, -o               Path to output JSON file
+                                                    [string] [default: "stdout"]
+  --delimiter, -d            Used CSV delimiter.                  [default: ","]
+  --multiValueDelimiter, -m  Used CSV delimiter in multiValue fields.
+                                                                  [default: ";"]
+  --continueOnProblems, -c   Flag if parsing should continue if module
+                             encounters an error.     [boolean] [default: false]
+  --logLevel                 Logging level: error, warn, info or debug
+                                                      [string] [default: "info"]
+  --prettyLogs               Pretty print logs to the terminal         [boolean]
+  --logFile                  Path to file where logs should be saved
+                                   [string] [default: "csv-parser-state.log"]
+```
+
+#### Info on flags
+- The `--projectKey` is only required if states have a `transitions` field containing one or more states that need to be resolved, otherwise it can be omitted. More info in the **Examples** below
+- The `--input` flag specifies the path to the states CSV file. If this flag is omitted, the module will attempt to read the data from the `standard input`.
+- The `--output` flag specifies where to output/save the parsed states as JSON file. Several notes on this flag:
+  - If the file specified already exists, it will be overwritten.
+  - The default location for status report logging is the standard output.
+  - If no output path is specified, the parsed states will be written to the standard output as a result, status reports will be logged to a `csv-parser-state.log` file in the current directory.
+- The `--delimiter` flag specifies the delimiter used in the input file. Defaults to `','` if omitted.
+- The `--multiValueDelimiter` flag specifies the delimiter for multiValue cells in CSV. Note that only the `transitions` and `roles` fields if present should contain multiple values. Defaults to `';'` if omitted.
+- The `--continueOnProblems` flag specifies if the module should continue parsing states if it encounters an error. Defaults to `false` if omitted.
```

Comment:
Is there another word to `continueOnProblems`.

Reply:
We had that discussion on previous PRs. By now it's established lingua. People liked it more than `--force` which would have been my preference.

Reply:
I would have preferred `--force` as well :)

Reply:
Me too. But for "usability" reasons, users of our tools prefer this